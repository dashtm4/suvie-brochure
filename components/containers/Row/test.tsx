import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Row } from '.'

describe('<Row />', () => {
  it('should renders children', () => {
    const text = 'test'

    const wrapper = shallow(<Row>{text}</Row>)

    expect(wrapper.text()).toMatch(text)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should render correct html with default props', () => {
    const text = 'test'

    const wrapper = shallow(<Row>{text}</Row>)

    expect(wrapper.html()).toMatch(
      `<div class="flex flex-wrap -mx-3 ">${text}</div>`
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should render correct html with not default className prop', () => {
    const text = 'test'

    const wrapper = shallow(<Row className="mb-10">{text}</Row>)

    expect(wrapper.html()).toMatch(
      `<div class="flex flex-wrap -mx-3 mb-10">${text}</div>`
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
