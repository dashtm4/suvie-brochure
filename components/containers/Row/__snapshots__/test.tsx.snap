// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should render correct html with default props 1`] = `
<div
  className="flex flex-wrap -mx-3 "
>
  test
</div>
`;

exports[`<Row /> should render correct html with not default className prop 1`] = `
<div
  className="flex flex-wrap -mx-3 mb-10"
>
  test
</div>
`;

exports[`<Row /> should renders children 1`] = `
<div
  className="flex flex-wrap -mx-3 "
>
  test
</div>
`;
